tests/unit/backend/layers/api/test_curation_api.py

```diff
@@ -632,12 +632,6 @@ def test__get_collection_verify_body_is_reshaped_correctly__OK(self, mock_config
         self.generate_dataset(
             collection_version=collection_version,
             metadata=dataset_metadata,
-            artifacts=[
-                DatasetArtifactUpdate(type="h5ad", uri="http://test_filename/1234-5678-9/local.h5ad"),
-                DatasetArtifactUpdate(type="rds", uri="http://test_filename/1234-5678-9/local.rds"),
-                DatasetArtifactUpdate(type="cxg", uri="http://test_filename/1234-5678-9/local.cxg"),
```

Comment:
minor: mind keeping the CXG update? I believe it's the only test where we confirm we filter out CXG artifacts, since the default generate_dataset doesn't include a CXG artifact.

Reply:
ah, good point!